code/game/objects/items/food/sandwichtoast.dm

```diff
@@ -259,23 +259,47 @@
 	. = ..()
 	obj_flags &= ~UNIQUE_RENAME // You shouldn't be able to disguise this on account of how it kills you
 
-///Override for checkliked callback
+// Makes you feel disgusted if you look at it wrong.
+/obj/item/food/sandwich/death/examine(mob/user)
+	. = ..()
+	if(check_liked(user) == FOOD_LIKED)
+		return
+	// Only human mobs, not animals or silicons, can be disgusted by this.
+	if(!istype(user, /mob/living/carbon/human))
+		return
```

Comment:
```suggestion
	// Only human mobs, not animals or silicons, can be disgusted by this.
	if(!ishuman(user))
		return
	if(check_liked(user) == FOOD_LIKED)
		return
```
1. `ishuman`, instead of `istype(..., human)`
2. Check human status before `check_liked`, or we'll runtime if a ghost examines us, for example.